pkg/virt-handler/migration-proxy/migration-proxy.go

```diff
@@ -403,6 +406,12 @@ func (m *migrationProxy) createUnixListener() error {
 		m.logger.Reason(err).Error("failed to create unix socket for proxy service")
 		return err
 	}
+	if m.nonRootLibvirtd {
+		if err := diskutils.DefaultOwnershipManager.SetFileOwnership(m.unixSocketPath); err != nil {
+			log.Log.Reason(err).Error("failed to change ownership on migration unix socket")
+			return err
+		}
```

Comment:
Is there any reason why we shouldn't do this unconditionally for nonroot and root VMIs? that way we don't have to pass all of this "nonRootLibvirtd" knowledge around.

Reply:
done